docs/core/tools/dotnet-test.md

```diff
@@ -50,7 +50,7 @@ List all of the discovered tests in the current project.
 
 `--filter <EXPRESSION>`
 
-Filters out tests in the current project using the given expression. For more information on filtering support, see [Running selective unit tests in Visual Studio using TestCaseFilter](https://aka.ms/vstest-filtering).
+Filters out tests in the current project using the given expression. For option details, see the [Filter option details](#filter-option-details) section. For a additional information and examples on how to use selective unit test filtering, see [Running selective unit tests](../testing/selective-unit-tests.md).
```

Comment:
"For more information" seems to me to be preferable to "For option details", since we're repeating "option details" in the title. 
a additional --> additional